### PR TITLE
OHRM5X-1663: Initiate admin - LDAP user synchronization

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue [hunk 1]
```diff
@@ -160,34 +160,49 @@
               </oxd-text>
             </oxd-grid-item>
             <oxd-grid-item class="--offset-row-4">
+              <oxd-input-field
+                v-model="configuration.userSearchFilter"
+                label="User Search Filter"
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue:165]: TODO:: i18n

Comment (Super-Chama) [reply src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue:165]: Will add a PR to your branch with these

Comment (Super-Chama) [reply src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue:165]: https://github.com/RajithaKumara/orangehrm/pull/48

Comment (RajithaKumara) [reply src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue]: Merged. Thanks :relaxed:

File: src/plugins/orangehrmLDAPAuthenticationPlugin/test/fixtures/testcases/LDAPConfigTestCases.yaml [hunk 2]
```diff
@@ -187,44 +214,45 @@ Update:
...
-        },
-        "groupObjectClass": "group",
-        "groupObjectFilter": "(&(objectClass=group)(cn=*))",
-        "groupNameAttribute": "cn",
-        "groupMembersAttribute": "member",
-        "groupMembershipAttribute": "memberOf",
-        "syncInterval": "60"
-      }
-    invalidOnly: [ "searchScope" ]
+# TODO
```
Comment (RajithaKumara) [inline src/plugins/orangehrmLDAPAuthenticationPlugin/test/fixtures/testcases/LDAPConfigTestCases.yaml:221]: TODO

**Pull request merged**